StaticAnalyzer/views/android/java.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf_8 -*-
+"""Module for java analysis."""
```

Comment:
We are not doing analysis here, we just list all the java files. Prolly this should be 
"list all java files"